t/image_upload_basics.t

```diff
@@ -0,0 +1,108 @@
+use strict;
+use warnings;
+
+# Database setup
+BEGIN {
+    unlink 'ddgc_test.db';
+    $ENV{DDGC_DB_DSN} = 'dbi:SQLite:dbname=ddgc_test.db';
```

Comment:
Yeah, anything you put in a `BEGIN` block is done first so you can set up the environment and such before any other code runs.